app/components/common/eventModal/index.tsx

```diff
@@ -51,133 +63,164 @@ export default function EventModal({
     return resultArray;
   };
 
+  const updateEventStateFromModal = (event: CalEvent) => {
+    setCalendarEvent((e) => ({ ...e, event }));
+  };
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    console.log(transition.state);
+    e.preventDefault();
+    // grab the form element
+    const $form = e.currentTarget;
+
+    // get the formData from that form
+    const formData = new FormData($form);
+
+    const payload = {
+      name: formData.get('title'),
+      startTime: dayjs(eventDetail?.start).valueOf(),
```

Comment:
What will dayjs(undefined) return? Will it work for .valuesOf()

Reply:
removed the `?` condition